lib/core/network/api_service.dart

```diff
@@ -139,4 +139,22 @@ class APIService {
       return jsonDecode(response.body) as Map<String, dynamic>;
     }
   }
+
+  Future<bool> sendRecEmail({required String id}) async {
```

Comment:
just a tiny style suggestion - avoid abbreviation in names, especially when a name abbreviation might have multiple meanings, like Rec, is it something about recording? :)